cmd/oras/root/push.go

```diff
@@ -183,6 +194,22 @@ func runPush(cmd *cobra.Command, opts *pushOptions) error {
 		}
 		desc.Annotations = packOpts.ConfigAnnotations
 		packOpts.ConfigDescriptor = &desc
+	} else if opts.Platform.Platform != nil {
+		blob, err := json.Marshal(opts.Platform.Platform)
+		if err != nil {
+			return err
+		}
+		desc := ocispec.Descriptor{
+			MediaType: oras.MediaTypeUnknownConfig,
```

Comment:
This value depends on the image-spec version.

Reply:
If the `image-spec` version is `v1.0`. The media type can also be the value of `--artifact-type`.

Reply:
Done.